beacon-chain/db/filesystem/metrics.go

```diff
@@ -38,7 +38,7 @@ func (bs *BlobStorage) Initialize() error {
 }
 
 // CollectTotalBlobMetric set the number of blobs currently present in the filesystem
-// to the blobsTotalGauge metric.
+// to the blobsWritten metric.
 func (bs *BlobStorage) collectTotalBlobMetric() error {
```

Comment:
I'm going to get rid of this and set up Initialize to instead run a no-op prune (slot=0) to warm up the cache. Because it's really slow on a cold cache, 4s vs 100ms - better to front-load that io at program start.